cleanlab/object_detection/filter.py

```diff
@@ -130,13 +136,18 @@ def _find_label_issues(
     if scoring_method == "objectlab":
         auxiliary_inputs = _get_valid_inputs_for_compute_scores(ALPHA, labels, predictions)
 
+        per_class_scores = get_per_class_ap(labels, predictions)
+        lab_list = [_separate_label(label)[1] for label in labels]
+        pred_list = [_separate_prediction(pred)[1] for pred in predictions]
+        pred_dict = _process_class_list(pred_list, per_class_scores)
+        lab_dict = _process_class_list(lab_list, per_class_scores)
```

Comment:
Isn't `lab_dict` a list? Isn't this just a list of lists of thresholds for every example given each class?

Reply:
https://github.com/cleanlab/cleanlab/blob/3024fad44f236c2f546610be35c3cfa5d011ce31/cleanlab/object_detection/filter.py#L143